scanners/ios/workflow.go

```diff
@@ -15,24 +15,29 @@ const (
 	AutomaticCodeSigningValue             = "api-key"
 	CacheLevelKey                         = "cache_level"
 	CacheLevelNone                        = "none"
-)
 
-const primaryTestDescription = `The workflow executes the tests. The *retry_on_failure* test repetition mode is enabled.`
+	// test workflow
+	primaryWorkflowID = "primary"
+
+	testWorkflowID          = "run_tests"
+	testWorkflowSummary     = "Run your Xcode tests and get the test report."
+	testWorkflowDescription = "The workflow will first clone your Git repository, cache and install your project’s dependencies if any, run your Xcode tests and save the test results."
 
-const primaryBuildOnlyDescription = `The workflow only builds the project because the project scanner could not find any tests.`
+	buildWorkflowID          = "build"
+	buildWorkflowSummary     = "Build your Xcode project."
+	buildWorkflowDescription = "The workflow will first clone your Git repository, cache and install your project’s dependencies if any and build your project."
 
-const primaryCommonDescription = `Next steps:
-- Check out [Getting started with iOS apps](https://devcenter.bitrise.io/en/getting-started/getting-started-with-ios-apps.html).
-`
+	// deploy workflow
+	deployWorkflowID = "deploy"
 
-const deployDescription = `The workflow tests, builds and deploys the app using *Deploy to bitrise.io* step.
+	archiveAndExportWorkflowID = "archive_and_export_app"
```

Comment:
Was there a discussion about the format of name? I cannot describe why but to me the name with `_` is harder to read then for example with `-`. You also used dashes in your PR name instead of underscores. And a couple of apps also replace spaces with dashes. I can only think about the Github desktop app but I am sure I have encountered more. 😄

So my question is what if we use dashes instead of underscores to separate the words in the new workflow names?